src/style.css

```
@@ -1 +1,12 @@
 /* styles go here */
+/* styles go here */
+.fieldset {
+  border: 1px solid;
+  margin-bottom: 21px;
```

```suggestion
  margin-bottom: 20px;
```
Чому така дивна цифра? Не користуйся "магічними числами"